tests/models/pix2struct/test_modeling_pix2struct.py

```diff
@@ -749,17 +749,20 @@ def test_batched_inference_image_captioning_conditioned(self):
         texts = ["A picture of", "An photography of"]
 
         # image only
-        inputs = processor(images=[image_1, image_2], text=texts, return_tensors="pt").to(torch_device)
+        inputs = processor(images=[image_1, image_2], text=texts, return_tensors="pt", add_special_tokens=False).to(
+            torch_device
+        )
 
         predictions = model.generate(**inputs)
 
         self.assertEqual(
-            processor.decode(predictions[0], skip_special_tokens=True), "A picture of a stop sign that says yes."
+            processor.decode(predictions[0], skip_special_tokens=True),
+            "A picture of a stop sign with a red stop sign on it.",
```

Comment:
Actually, seeing [this image](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/australia.jpg), I would say this new prediction is better than the previous one.